public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -133,6 +135,19 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
     loadCompOptions(fromVersion, toVersion)
   }
 
+  function updateChartContext(grid) {
+    // Update context chart data for given rendered rows
+    // sets [min, max] list to the scope for visible rows
+    vm.refCtx = d3.extent(
+      _.reduce(grid.renderContainers.body.renderedRows, function(m, d) {
+        return m.concat([
+          Math.log(d.entity.avgVsSelf[0]),
+          Math.log(d.entity.avgVsSelf[1])
+        ])
```

Comment:
Are d.entity.avgVsSelf values of 0 possible?  Otherwise -Infinity will swamp out the lower bound?

Reply:
Normally - no. Charts don't handle such situation either.